dask/dataframe/groupby.py

```diff
@@ -179,79 +318,42 @@ class SeriesGroupBy(_GroupBy):
 
     _token_prefix = 'series-groupby-'
 
-    def __init__(self, df, index, key=None, **kwargs):
-        self.df = df
-        self.index = index
-        self.key = key
-        self.kwargs = kwargs
+    def __init__(self, df, index, slice=None, **kwargs):
 
+        # raise pandas-compat error message
         if isinstance(df, Series):
+            # When obj is Series, index must be Series
             if not isinstance(index, Series):
-                raise TypeError("A dask Series must be used as the index for a"
-                                " Series groupby.")
-            if not df.divisions == index.divisions:
-                raise NotImplementedError("The Series and index of the groupby"
-                                          " must have the same divisions.")
+                if isinstance(index, list):
+                    if len(index) == 0:
+                        raise ValueError("No group keys passed!")
+                    msg = "Grouper for '{0}' not 1-dimensional"
+                    raise ValueError(msg.format(index[0]))
+                # raise error from pandas
+                df._pd.groupby(index)
+        super(SeriesGroupBy, self).__init__(df, index=index,
+                                            slice=slice, **kwargs)
 
     @property
     def column_info(self):
-        return self.key
-
-    def apply(self, func, columns=None):
-        """ Apply function to each group.
-
-        If the grouper does not align with the index then this causes a full
-        shuffle.  The order of rows within each group may not be preserved.
-        """
-        # df = set_index(self.df, self.index, **self.kwargs)
-        if self.index._name == self.df.index._name:
-            return map_partitions(_groupby_level0_getitem_apply,
-                                  self.df, self.key, func,
-                                  columns=columns)
-        else:
-            from .shuffle import shuffle
-            df = shuffle(self.df, self.index, **self.kwargs)
-            return map_partitions(_groupby_apply,
-                                  columns or self.df.columns,
-                                  self.df, self.index, func)
+        raise ValueError('temp, deprecate')
```

Comment:
In other places a warning is thrown for deprecated `column_info` - is there a reason to not do that here as well? https://github.com/dask/dask/blob/master/dask/dataframe/core.py#L945